.gitignore

```diff
@@ -1 +1,2 @@
 vendor/
+.idea/
```

Comment:
@tomphp no problem. Usually I prefer on local as easily "transportable" but I can surely remove this.